rover/src/owr_usb_reset/scripts/usb_reset.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# license removed for brevity
+import rospy
+import subprocess
+from std_msgs.msg import String,Int16
+from owr_messages.msg import devices
+
+"""in visudo, need to add %ros ALL = NOPASSWD: /home/ros/owr_software/rover/src/owr_usb_reset/usb_reset
+after %sudo	ALL=(ALL:ALL) ALL. This allows the usb reset program to run without needing a password for 
+sudo. Eneter password manually while running causes a problem 
+"""
+
+def idmap(device):
+	if (device[23:27] == "1d6b"):
+		return "linux"
+	else:
+		return device[0:32]
+
+def callback(data):
+	subprocess.call("cd /dev/bus/usb",shell=True)
+	devs = subprocess.check_output(["lsusb"])
+	devs = devs.split("\n")
+	if(data.data < len(devs) - 1): 
+		devs[data.data].replace(' ', '')
+		print(devs[data.data])
+		filepath = "/dev/bus/usb/" + devs[data.data][4:7]
+		filepath = filepath + "/" + devs[data.data][15:18]
+		print(filepath)
```

Comment:
please use rospy.loginfo logging rather than prints (this allows us to view and controls logs from the ros console)